.gitmodules

```
@@ -58,3 +54,15 @@
 	path = third-party/wlr-protocols
 	url = https://gitlab.freedesktop.org/wlroots/wlr-protocols.git
 	branch = master
+[submodule "third-party/nvenc-headers/1100"]
+	path = third-party/nvenc-headers/1100
+	url = https://github.com/FFmpeg/nv-codec-headers.git
+	branch = sdk/11.0
+[submodule "third-party/nvenc-headers/1200"]
+	path = third-party/nvenc-headers/1200
+	url = https://github.com/FFmpeg/nv-codec-headers.git
+	branch = sdk/12.0
+[submodule "third-party/nvenc-headers/1202"]
+	path = third-party/nvenc-headers/1202
+	url = https://github.com/FFmpeg/nv-codec-headers.git
+	branch = master
```

```suggestion
[submodule "third-party/nvenc-headers/1100"]
	path = third-party/nvenc-headers/1100
	url = https://github.com/FFmpeg/nv-codec-headers.git
	branch = sdk/11.0
[submodule "third-party/nvenc-headers/1200"]
	path = third-party/nvenc-headers/1200
	url = https://github.com/FFmpeg/nv-codec-headers.git
	branch = sdk/12.0
[submodule "third-party/nvenc-headers/1202"]
	path = third-party/nvenc-headers/1202
	url = https://github.com/FFmpeg/nv-codec-headers.git
	branch = master
```

Can you place these above `third-party/nvapi-open-source-sdk`

---

Done